app/views/environments/welcome.html.erb

```diff
@@ -11,4 +11,7 @@
   <div class="blank-slate-pf-main-action">
       <%= new_link(_('New Puppet environment'), {}, { :class => "btn-lg" }) %>
   </div>
+  <div class="blank-slate-pf-secondary-action">
+    <%= select_action_button( _('Import'), {}, import_proxy_links(hash_for_import_environments_environments_path, 'btn btn-default')) %>
```

Comment:
@tbrisker - You are right. Haven't used it in the first place because I have changed the css classes. But after @dLobatog's comment reverted back the css classes.
Anyhow, changed to `import_proxy_select`. Thanks